packages/react/src/components/Accordion/AccordionItem.js

```diff
@@ -41,6 +42,8 @@ function AccordionItem({
     [customClassName]: !!customClassName,
   });
 
+  const Toggle = renderToggle || renderExpando; // remove renderExpando in next major release
```

Comment:
I think this logic may result in a breaking change if the `renderToggle` prop above has a default value.

Specifically, since `renderToggle` is _always_ defined it will always be used instead of `renderExpando`. As a result, `renderExpando` would not be used.

I think to fix it we would need to remove one of the optional values, probably:

```
  renderExpando = defaultRenderToggle, // remove renderExpando in next major release
  renderToggle,
```

So that `renderToggle` is used if it's defined, otherwise it will fallback to `renderExpando` and support existing usage.

Let me know if this makes sense!

Reply:
Ok, i removed the default prop to `renderToggle`.

Reply:
I'm worried that `renderToggle` won't receive the goodies from `defaultRenderToggle` now. Should I be worried?

Reply:
@sstrubberg this expressions: `renderToggle || renderExpando` ends up looking like:

- Is `renderToggle` defined?
  - Yes, then use it
  - No, fallback to `renderExpando`

`renderExpando` evaluates to either a custom value, if it's provided as a prop, but by default it is `defaultRenderToggle`

As a result, the fallback is still `defaultRenderToggle`, and folks are able to use either `renderToggle` or `renderExpando` 👍